src/lib/WIFI/devWIFI.cpp

```diff
@@ -414,6 +415,20 @@ static void WebUpdateModelId(AsyncWebServerRequest *request)
   request->client()->close();
   rebootTime = millis() + 100;
 }
+
+static void WebUpdateForceTelemetry(AsyncWebServerRequest *request)
+{
+  long forceTlm = request->arg("force-tlm").toInt();
+
+  DBGLN("Setting force telemetry %u", (uint8_t)forceTlm);
+  config.SetForceTlmOff(forceTlm != 0);
+  config.Commit();
+
+  AsyncWebServerResponse *response = request->beginResponse(200, "text/plain", "Force telemetry updated");
```

Comment:
This can just be `request->send(200, "text/plain", "Force telemetry updated");` without the code that follows it (no response variable, headers, send, close, none of that, just what I've got here)

Reply:
I did try to remove the code but then the frontend shows an error message since the request does not have a response